cmd/main.go

```diff
@@ -257,6 +261,45 @@ func main() {
 	c := make(chan os.Signal, 1)
 	signal.Notify(c, os.Interrupt)
 
+	go func() {
+		kclient, err := mesherykube.New(nil)
+		if err != nil {
+			log.Error(err)
+		}
+		mesheryclient, err := operatorClient.New(&kclient.RestConfig)
+		if err != nil {
+			if mesheryclient == nil {
+				log.Error(model.ErrMesheryClient(nil))
+			}
+			log.Error(model.ErrMesheryClient(err))
+		}
+		broker, err := mesheryclient.CoreV1Alpha1().Brokers(MesheryNamespace).Get(context.Background(), "meshery-broker", metav1.GetOptions{})
```

Comment:
`context.Background()` would be almost pointless I think? Let's either propagate from parent or let's use `context.TODO()`?

Reply:
Yep. Makes sense 👍